tradeexecutor/statistics/core.py

```diff
@@ -192,12 +188,17 @@ def update_statistics(
         Only available when `update_statistics()` is run
         at the end of live trading cycle.
     """
-
     logger.info(
         "update_statistics(), real-time clock at %s, strategy cycle at %s",
         clock,
         strategy_cycle_or_wall_clock
     )
+    
+    logger.info("Serialising serialise_long_short_stats_as_json_table()")
```

Comment:
Only write a log message if you are within the `if` condition. Otherwise the log message is misleading, causing problems if someone is trying to diagnose any issues.